core/src/test/java/org/commonjava/maven/ext/core/fixture/TestUtils.java

```diff
@@ -175,9 +194,9 @@ public static Object executeMethod( Object instance, String name, Class<?>[] typ
         @Getter
         private final MavenExecutionRequest request;
 
-        private SMContainer( MavenExecutionRequest req, ManipulationSession session, ManipulationManager manager )
+        private SMContainer( MavenExecutionRequest request, ManipulationSession session, ManipulationManager manager )
```

Comment:
Now that the pom _can be_ passed to `createSessionAndManager` I don't think the request needs to be accessible via the `SMContainer`. While this is used in PME tests and in the groovy-example project I think this API change would make things simpler.

Reply:
After this API s finalized, I will submit a pull request to change the examples (if necessary).

Reply:
I think we're talking at cross-purposes - even within _here_ I don't think the request needs to be accessible due to your preceding changes?

Reply:
I made the change to `SMContainer` since `request` is no longer needed to set the pom as you said. Still, I think we need a better solution for when there is no pom.